ddtrace/internal/utils/cache.py

```diff
@@ -114,12 +114,12 @@ def callonce(f):
 
     def _():
         # type: () -> Any
+        exc = None
```

Comment:
Why is this moved?

Reply:
That error is caused by the `if`. I don't think it is currently possible for `exc` to cause this exception. So I think this change can be reverted

Reply:
You are right, I can't see a point where `exc` is not defined if the if is removed, restored.